src/twisted/internet/test/test_inlinecb.py

```diff
@@ -150,7 +152,33 @@ def calling():
         self.assertIn("in calling", tb)
         self.assertIn("Error Marker", tb)
 
-    def test_forwardLotsOfTracebacks(self):
+    @skipIf(sys.version_info < (3, 13), "Needs Python 3.13 or newer")
+    def test_forwardTracebacks_313(self):
+        """
+        Chained inlineCallbacks are forwarding the traceback information
+        from generator to generator.
+
+        A first simple test with a couple of inline callbacks for Python 3.13 and later.
+        """
+
+        @inlineCallbacks
+        def erroring():
+            yield "forcing generator"
+            raise Exception("Error Marker")
+
+        @inlineCallbacks
+        def calling():
```

Comment:
this needs to be called `calling1` or something more specific.
Otherwise `self.assertIn("in calling", tb)` will match `in callingANYTHIG`

and better to use ``self.assertIn("in calling1\n", tb)` to make sure no extra text is added

Reply:
This is the single callback test.